### PR TITLE
Add a table of contents entry for Machine ID v14 migration

### PR DESCRIPTION
**Pull request by ptgott**
The Machine ID upgrade guide is missing a table of contents entry. Add one for it.

### CODE REVIEW

**Comment by ptgott** [comment]
Only applies to the v14 docs!

**Review by zmb3 — APPROVED** [review]

**Review by stevenGravy — APPROVED** [review]

**Comment by ptgott** [comment]
Going to wait for a review from @strideynet.

**Review by strideynet — APPROVED** [review]
thanks for catching this!

**Pull request merged**
